messages/lightning.preview.app.md

```diff
@@ -4,7 +4,7 @@ Preview Lightning Experience Applications.
 
 # description
 
-Preview components, org, and sites. If no topic is specified, the default action is to preview the org.
+Preview components, apps, and sites. If no topic is specified, the default action is to preview the org.
```

Comment:
A couple of things on:

> If no topic is specified, the default action is to preview the org.

- Is there precedent for this? Having default behavior at the topic hierarchy level? I personally find that confusing, based on the fact that the different subtopics have different input parameters. We should confirm.
- If that behavior is confirmed, we need to change the default action verbiage to something like "...to preview the most recently accessed Lightning App".

Reply:
Oh wait, I'm realizing that this is verbiage *specifically* for `sf lightning preview app`. It's the *beginning of the description* that threw me off. I'll make an edit suggestion.

Reply:
Ah I see that these values have already been defined in the spec. This one just has the wrong verbiage. Should be:

```suggestion
Preview Lightning Experience Applications in real-time.
```